packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -30,13 +30,28 @@ BuildRequires: puppet-agent-puppet-strings < 2
 %description
 Complete installer for The Foreman life-cycle management system based on Puppet.
 
+%package katello
+Summary: Katello installer bits
```

Comment:
Maybe we should update this to something like `Katello scenario for Foreman Installer`

Reply:
Good point. I'll make it plural since it also contains a proxy scenario. In the future I hope to add more, like pulp and candlepin standalone.